platform/web/detect.py

```diff
@@ -38,8 +38,7 @@ def get_opts():
         BoolVariable("use_safe_heap", "Use Emscripten SAFE_HEAP sanitizer", False),
         # eval() can be a security concern, so it can be disabled.
         BoolVariable("javascript_eval", "Enable JavaScript eval interface", True),
-        BoolVariable("threads_enabled", "Enable WebAssembly Threads support (limited browser support)", True),
-        BoolVariable("gdnative_enabled", "Enable WebAssembly GDNative support (produces bigger binaries)", False),
+        BoolVariable("dlink_enabled", "Enable WebAssembly dynamic linking (GDExtension support). Produces bigger binaries", False),
```

Comment:
`black` wants this split on 3 lines because it's longer than 120 char (you can apply it with `black -l 120`).